examples/js/loaders/sea3d/SEA3DLoader.js

```diff
@@ -25,12 +25,12 @@ THREE.ShaderLib.replaceCode = function(src, target, replace) {
 THREE.ShaderLib['phong'].fragmentShader_StandardMaterial = 
 	THREE.ShaderLib.replaceCode( THREE.ShaderLib['phong'].fragmentShader, [
```

Comment:
I did not realize this was in your previous version, too.

Why do you feel it is necessary to modify the `Phong` shader code in the sea3d loader?

I notice you set `material.emissive.set( 0.2, 0.2, 0.2 )` (or something similar) on almost all your models.

You appear to want `emissive` to be modulated by both `material.color` and `material.map`.

In the `Phong` shader, `material.emissive` is simply additive to the reflected color.